package.json

```diff
@@ -21,7 +21,11 @@
   },
   "homepage": "https://github.com/Financial-Times/n-internal-tool#readme",
   "dependencies": {
-    "@financial-times/n-express": "^19.21.4"
+    "@financial-times/n-express": "^19.21.4",
+    "dateformat": "^3.0.3",
+    "denodeify": "^1.2.1",
```

Comment:
as of 8.x denodify can be replaced by promisify in node.js core: https://nodejs.org/dist/latest-v8.x/docs/api/util.html